libcontainer/configs/intelrdt.go

```diff
@@ -10,4 +10,12 @@ type IntelRdt struct {
 	// The unit of memory bandwidth is specified in "percentages" by
 	// default, and in "MBps" if MBA Software Controller is enabled.
 	MemBwSchema string `json:"memBwSchema,omitempty"`
+
+	// The flag to indicate if Intel RDT CMT is enabled. CMT ( Cache Monitoring Technology ) supports monitoring of
```

Comment:
extra spaces within `()`

IOW
```suggestion
	// The flag to indicate if Intel RDT CMT is enabled. CMT (Cache Monitoring Technology) supports monitoring of
```